synapse/models/gov/intl.py

```diff
@@ -8,6 +8,9 @@ def getModelDefs(self):
                 ('iso:oid', ('str', {'regex': '^([0-2])((\.0)|(\.[1-9][0-9]*))*$'}), {
                     'doc': 'An ISO Object Identifier string.'}),
 
+                ('iso:3166:cc', ('str', {'lower': True, 'regex': '^[a-z]{2}$'}), {
```

Comment:
How is this different from ``pol:iso2`` ?

Reply:
pol:iso2 is a terribly mis-named type that i would like us to move away from.  the first step is to have something else to move to. :)